notification-transport/src/main/java/org/hypertrace/alert/engine/notification/transport/NotificationSenderConfig.java

```diff
@@ -0,0 +1,28 @@
+package org.hypertrace.alert.engine.notification.transport;
+
+import com.typesafe.config.Config;
+
+public class NotificationSenderConfig {
+  private static final String NOTIFICATION_CONFIG = "notification.config";
+  private static final String TRACEABLE_URL = "traceable.url";
```

Comment:
seems specific url?

Reply:
actually this config file is no where used will remove it